src/components/FormField/FormField.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+import { FormFieldStyled, ErrorSpan, Label, TextArea } from "components/FormField/FromField.styled";
+
+export interface FormFieldProps {
+  field: any;
+  form: {touched: any, errors: any};
+  type: string;
+  placeholder: string;
+  step?: number
+}
+
+const FormField: React.FC<FormFieldProps> = (props) => {
+  return (
+    <>
+      <Label htmlFor={props.field.name}>{props.field.name}</Label>
+      {props.type === 'textarea' ?
+        <TextArea
+          placeholder={props.placeholder}
+          {...props.field}
+          rows={4}
+          cols={50}  /> :
```

Comment:
Something happened to formatting again, very difficult to read until you realize it's just not formatted. I highly recommend to figure how to make your IDE to format code on file save.